src/components/post-comment.jsx

```diff
@@ -0,0 +1,51 @@
+import { useState } from 'react';
+import { Form ,Button,Card} from 'react-bootstrap';
+import {insertComment} from '../api'
+export const PostComment = ({id,setComments}) => {
+    const [Comment, setNewComment] = useState("");
+    const [err,setErr]=useState(false)
+    const [posted , setPosted] = useState(null)
+  
+    const handleSubmit = (event) =>{
+        
+      event.preventDefault()
+      
+
+        const commentForApi={
+            username: 'cooljmessy',
+            body: Comment
+        }
+        setPosted(true)
+        insertComment(id,commentForApi).then(()=>{
+            setPosted(null)
+            setNewComment("")
+            window.location.reload()
```

Comment:
you shouldn't be reloading the page

Reply:
you don't want to refetch all of the date. Just take the inserted comment and add it into state locally